app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -57,19 +57,24 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
 
 If you are interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for a single payment of £10,000 from the UK government. [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment). 
 
-## Apply for qualified teacher status (QTS)
+## English qualified teacher status (QTS)
 
 Qualified teacher status (QTS) is the professional status teachers in England gain at the end of their teacher training. Schools use QTS as a measure of teaching skills and experience, so you will find it much easier to get a teaching job in England if you have QTS.
 
 However, if you’re a qualified teacher from outside the UK, you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#qts-exemption-for-teachers-from-outside-the-uk).
 
 After that, you will need QTS to teach in many schools in England, although it is not a legal requirement in some types of school (for example, [academy schools, free schools and private schools](https://www.gov.uk/types-of-school)). 
+ 
 
-It’s important to note that, even if you decide to apply for QTS, you will have to apply for your teaching job and your visa separately. 
+### Applying for QTS
 
-### How to apply for QTS
+Depending on where you qualified and the subject you teach, you may be able to use a new service to check if you meet the requirements for English QTS and apply.  
 
-If you have a teaching qualification from the following countries and regions, you can [check if you meet the requirements for QTS and apply](https://apply-for-qts-in-england.education.gov.uk/eligibility/start):
+You will not have to pay a fee or undergo further training. However, you will have to meet strict requirements. These requirements are mandatory and must be met in full exactly as set out in [‘Awarding qualified teacher status to overseas teachers’](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers).  
+
+It's important to note that an award of QTS does not lead to a teaching job or visa – you will need to apply for these separately. 
+
+If you have a teaching qualification from the following countries and regions, you can check if you meet the requirements for QTS and apply:
 
```

Comment:
Is there a reason we say countries and regions? Could it just be countries?

Reply:
the EEA is a collection of countries rather than a country, was my thinking